packages/opencensus-web-core/src/trace/model/tracing.ts

```diff
@@ -39,6 +39,12 @@ export class Tracing implements webTypes.Tracing {
     return this.singletonInstance || (this.singletonInstance = new this());
   }
 
+  constructor() {
+    // register the noop exporter to have an span event listener,
+    // helpful for testing.
+    this.tracer.registerSpanEventListener(this.exporter);
```

Comment:
Could you do this in the tests rather than here in the production code?

Reply:
Done